zipkin-storage/zipkin2_cassandra/src/main/java/zipkin2/storage/cassandra/CassandraSpanStore.java

```diff
@@ -73,6 +76,15 @@
     }
   }
 
+  private static SelectTraceIdsFromSpan.Factory initialiseSelectTraceIdsFromSpan(Session session) {
```

Comment:
maybe add a comment here with the same rationale as the PR desc (which this allows dropping the SASI fields)

Reply:
done.